projects/assets/src/translations/en/common.ts

```diff
@@ -133,6 +133,8 @@ export const common = {
     },
     cartNotFound: 'Cart not found.',
     invalidCodeProvided: 'Invalid code provided.',
+    unknownIdentifier: 'Item not found',
```

Comment:
Should the message end with a dot? (most of them seem to have one)

Reply:
Item as in "cart item" or as any kind of item not found?

Reply:
any kind of item.